docs/common-bugs.rst

```diff
@@ -0,0 +1,278 @@
+
+Storage issues
+--------------
+
+These issues are related to hardware, partitioning and storage configuration.
+
+Bug in blivet
+^^^^^^^^^^^^^
+
+:Detection: The exception starts in ``blivet`` or ``libblockdev``.
+:Solution: It seems to be an issue in the storage configuration library. Reassigning to blivet.
+:Example: `rhbz#1827254 <https://bugzilla.redhat.com/show_bug.cgi?id=1827254>`_
+
+Bug in blivet-gui
+^^^^^^^^^^^^^^^^^
+
+:Detection: The exception starts in ``blivet-gui`` or there is a problem with partitioning and
+    the reporter used Blivet-GUI as the partitioning method.
+:Solution: It seems to be an issue in blivet-gui. Reassigning.
+:Example: `rhbz#1833775 <https://bugzilla.redhat.com/show_bug.cgi?id=1833775>`_
+
+Failing hardware
+^^^^^^^^^^^^^^^^
+
+:Detection: The logs (journal or syslog) are full of kernel messages about I/O errors. For
+    example::
+
+        kernel: [sdb] tag#9 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_SENSE
+        kernel: [sdb] tag#9 Sense Key : Medium Error [current]
+        kernel: [sdb] tag#9 Add. Sense: Unrecovered read error - auto reallocate failed
+        kernel: [sdb] tag#9 CDB: Read(10) 28 00 1d 04 10 00 00 00 08 00
+        kernel: print_req_error: I/O error, dev sdb, sector 486805504
+
+:Solution: It looks like a hardware failure. Please, check your hardware.
+:Example: `rhbz#1633457 <https://bugzilla.redhat.com/show_bug.cgi?id=1633457>`_
+
+Unlocked LUKS
+^^^^^^^^^^^^^
+
+:Detection: The storage configuration fails with a message ``luks device not configured``.
+:Solution: Anaconda doesn't support LUKS devices that are unlocked outside the installer. The
+    device has to be unlocked in Anaconda.
+:Example: `rhbz#1624617 <https://bugzilla.redhat.com/show_bug.cgi?id=1624617>`_
+
+Cannot toggle container encryption
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: It is not possible to toggle the container encryption in the custom partitioning spoke.
+:Solution: Duplicate of the bug 1827254.
+:Example: `rhbz#1827254 <https://bugzilla.redhat.com/show_bug.cgi?id=1827254>`_
+
+Undetected partitions
+^^^^^^^^^^^^^^^^^^^^^
+
+:Detection: When the custom partitioning spoke is entered, it raises an exception with a message:
+    ``cannot initialize a disk that has partitions``. Anaconda tries to initialize disks that are
+    supposed to be empty, but there are partitions that were not discovered by kernel after boot.
+:Solution: Duplicate of the bug 1825067.
+:Example: `rhbz#1825067 <https://bugzilla.redhat.com/show_bug.cgi?id=1825067>`_
+
+Bootloader issues
+-----------------
+
+There issues are related to bootloader issues.
+
+Bug in bootloader
+^^^^^^^^^^^^^^^^^
+
+:Detection: The exception is raised during a bootloader installation with a message that usually
+    says ``failed to write bootloader`` or ``boot loader install failed``. Look into the
+    ``program.log`` for more information.
+:Solution: Could the bootloader team have a look at this bug, please?
```

Comment:
It would be better to explicitly say to re-assign the reported bug to the bootloader team or set needinfo on the bootloader team.
People fighting with a bootloader problem and finding this document could start spamming the bootloader mailing list with requests for help. I would do it after reading the solution :-)

Reply:
I have removed the contact.